### PR TITLE
Update Rubocop ~> 0.49.0

### PR DESCRIPTION
**Pull request by JulienItard**
There are tons of new options. Maybe good to check them.

### CODE REVIEW

File: bookingsync-stylecheck.gemspec [hunk 1]
```diff
@@ -10,6 +10,6 @@ Gem::Specification.new do |s|
   s.homepage      = "http://gihub.com/BookingSync/bookingsync-stylecheck"
   s.license       = "MIT"
 
-  s.add_dependency "rubocop", "~> 0.46.0"
+  s.add_dependency "rubocop", "~> 0.49.0"
```
Comment (thilonel) [inline bookingsync-stylecheck.gemspec:13]: Actually, why do we lock this here? Isn't the `Gemfile.lock` enough?

File: config/rubocop.yml [hunk 2]
```diff
@@ -204,9 +204,15 @@ Style/TrailingBlankLines:
...
 Lint/EndAlignment:
-  AlignWith: variable
+  EnforcedStyleAlignWith: variable
   Enabled: true
+
+Style/SingleLineBlockParams:
+  Enabled: true
+
+Style/SymbolArray:
+  Enabled: false
```
Comment (thilonel) [inline config/rubocop.yml:218]: So first 2 got renamed, you got that from the changelog?
What about this SingleLineBlockParams and SymbolArray?
A dependency updating commit shouldn't introduce new rules, although that's the only thing stated in the changelog. So what are these? :)

Comment (JulienItard) [reply config/rubocop.yml:218]: @thilonel 
All from Changelog, 2 first renamed yes.
2 news changed their default. That's why i need to adapted it to keep exact same rules than previous

Comment (thilonel) [reply config/rubocop.yml:218]: Good job! :)

**Review by thilonel — APPROVED** [review]
LGTM, but I still think we could remove the version limit from the Gemspec on rubocop.

**Comment by Mandaryn** [comment]
and if they bump rubocop with uncompatibile changes like those then what? lock it in apps gemfile? here is the right place to define the dependency version we rely on

**Comment by thilonel** [comment]
That's the correct answer, thank you Manda.

**Comment by JulienItard** [comment]
You lost me :) What am I suppose to do ?

**Pull request merged**